src/core/positioning/positioningsource.cpp

```diff
@@ -150,9 +150,37 @@ void PositioningSource::setBackgroundMode( bool backgroundMode )
 
   mBackgroundMode = backgroundMode;
 
+  if ( mBackgroundMode )
+  {
+    QFile file( QStringLiteral( "%1.information" ).arg( backgroundFilePath ) );
+    file.open( QFile::WriteOnly );
+    file.close();
```

Comment:
Is this done in order to truncate the file? Should this be done at the end of `getBackgroundPositionInformation` instead? Should `QIODeviceBase::Truncate` be used instead?